CMakeLists.txt

```diff
@@ -30,41 +43,32 @@ if(BUILD_TESTING)
   add_subdirectory(test)
 endif()
 
-#
-# mention the library target as export library
-export(TARGETS double-conversion
-  FILE "${PROJECT_BINARY_DIR}/double-conversionLibraryDepends.cmake")
-
-#
-# set this build as an importable package
-export(PACKAGE double-conversion)
+install(TARGETS double-conversion EXPORT double-conversionTargets
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+    RUNTIME DESTINATION bin
+    INCLUDES DESTINATION include)
 
-#
-# make a cmake file -- in this case, all that needs defining
-# is double-conversion_INCLUDE_DIRS
-configure_file(double-conversionBuildTreeSettings.cmake.in
-  "${PROJECT_BINARY_DIR}/double-conversionBuildTreeSettings.cmake"
-  @ONLY)
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
```

Comment:
If you will use `project(... VERSION X.Y.Z)` there is no need to add `VERSION` here, `PROJECT_VERSION` will be used automatically.

Reply:
Good to know.